packages/create-sitecore-jss/src/templates/nextjs-personalize/src/components/CdpPageView.tsx

```diff
@@ -72,10 +71,12 @@ const CdpPageView = (): JSX.Element => {
     if (disabled()) {
       return;
     }
+
+    const siteInfo = siteResolver.getByName(siteName);
```

Comment:
Any reason to not combine the `siteName` line from above here as well (so it's all in one place)? And update the useEffect dependency to `site`...
```suggestion
    const siteInfo = siteResolver.getByName(site?.name || config.jssAppName);
```